crates/bindings-csharp/Runtime/Internal/FFI.cs

```diff
@@ -229,4 +271,20 @@ internal static class ConsoleTimerIdMarshaller
 
     [LibraryImport(StdbNamespace)]
     public static partial CheckedStatus console_timer_end(ConsoleTimerId stopwatch_id);
+
+    [LibraryImport(StdbNamespace)]
+    public static partial CheckedStatus iter_start_filtered(
```

Comment:
Looks like a rebase issue. This ABI was removed and shouldn't be used anymore.

Reply:
Same as with `bindings.c`, maybe worth reverting the reordering here to make it obvious what's actually added and what is an accidental regression.